### PR TITLE
Add ScottoWing keyboard

### PR DESCRIPTION
**Pull request by joe-scotto**
## Description

Adds the ScottoWing (PCB Edition) keyboard

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/scottokeebs/scottowing/info.json [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+            "GP4",
+            "GP5",
+            "GP6",
+            "GP7",
+            "GP20",
+            "GP22",
+            "GP26",
+            "GP27",
+            "GP28",
+            "GP29"
```
Comment (waffle87) [inline keyboards/scottokeebs/scottowing/info.json:28]:
```suggestion
            "GP2", "GP3", "GP4", "GP5", "GP6", "GP7", "GP20", "GP22", "GP26", "GP27", "GP28", "GP29"
```

File: keyboards/scottokeebs/scottowing/keymaps/default/config.h [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+#pragma once
+
+// Define options
+#define TAPPING_TERM 135
+#define PERMISSIVE_HOLD
+#define TAPPING_TERM_PER_KEY
+
+// Bootmagic
+#define BOOTMAGIC_LITE_ROW 0
+#define BOOTMAGIC_LITE_COLUMN 1
```
Comment (waffle87) [inline keyboards/scottokeebs/scottowing/keymaps/default/config.h:27]: This is not something that should be applied at the keymap level. If it is needed/intended, it should be placed at the keyboard level `info.json`.
```suggestion
```

File: keyboards/scottokeebs/scottowing/readme.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
+# ScottoWing (PCB Edition)
+
+![ScottoWing](https://i.imgur.com/ckaYMAxh.jpeg)
```
Comment (waffle87) [inline keyboards/scottokeebs/scottowing/readme.md:3]:
```suggestion
![ScottoWing](https://i.imgur.com/ckaYMAx.jpeg)
```

**Review by waffle87 — CHANGES_REQUESTED** [review]

File: keyboards/scottokeebs/scottowing/keymaps/default/config.h (hunk 2 above)
Comment (joe-scotto) [reply keyboards/scottokeebs/scottowing/keymaps/default/config.h]: This is needed for this board, added it to the `info.json` in [63d994](https://github.com/qmk/qmk_firmware/pull/23513/commits/263d99449e875e46419ced1bbb1d758a2999aa3e)

File: keyboards/scottokeebs/scottowing/info.json [hunk 4]
```diff
@@ -0,0 +1,76 @@
+{
+    "manufacturer": "ScottoKeebs",
+    "keyboard_name": "ScottoWing (PCB Edition)",
+    "maintainer": "joe-scotto",
+    "bootloader": "rp2040",
+    "bootmagic": {
+        "enabled": true,
```
Comment (waffle87) [inline keyboards/scottokeebs/scottowing/info.json:7]:
```suggestion
```

**Review by waffle87 — APPROVED** [review]
Otherwise lgtm.

**Review by drashna — APPROVED** [review]

**Pull request merged**